cancellation/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "vscode-languageserver-cancellation",
```

Comment:
May be a better name is vscode-lsp-file-based-cancellation to make it more clear that is is about file based cancellation

Reply:
so, a question I had was whether I will make 1 npm for 1 custom cancellation package or I will have 1 cancellation package with multiple custom cancellation strategy.